test/parallel/test-tls-client-reject.js

```diff
@@ -35,6 +35,8 @@ const options = {
 
 const server = tls.createServer(options, function(socket) {
   socket.pipe(socket);
+  // Pipe already ends... but leaving this here tests .end() after .end().
+  // XXX(sam) is that good, or is it bad?
   socket.on('end', () => socket.end());
```

Comment:
I think it’s just odd, tbh. If we have stream/net tests covering this we can drop it, imo.

Reply:
hard to weed through all the tests to see if its covered, I'll leave it.